deployment/hasura/migrations/Aerie/5_sequence_adaptation_name/down.sql

```diff
@@ -1,5 +1,5 @@
 alter table sequencing.sequence_adaptation
-  drop constraint sequence_adaptation_natural_key,
+  drop constraint sequence_adaptation_unique_key,
```

Comment:
Oh I was saying in the thread that the usual naming pattern would be like `sequence_adaptation_name_unique`